specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,264 @@
+# Background
+
+We are exposing an event that will fire when an attempt to launch a registered protocol is made. The host will be given the option to cancel the launch, handle the popup dialog to hide the dialog, as well as revoke permissions that have been previously given to the origin launching the protocol. 
```

Comment:
Why are these called registered protocols - does that match the naming from somewhere else? Since these aren't protocols registered with WebView2, maybe we should be clearer in the name like `system registered protocol` or `OS registered protocol` or something?

In any case, this Background section needs to define what a registered protocol is and what it means to launch it.

Reply:
I see in the other PR that this is named for Windows docs on app protocols. Still, naming this `Registered` Protocols is confusing because at least we'll have custom URI schemes that WebView2 knows about so just calling these `registered` isn't enough to identify what we're talking about. Even calling them `App` protocols is sort of ambiguous since its not clear that we mean (host) app vs WebView2. Although calling it OS or System is sort of incorrect since its another app that's handling the URI not the OS exactly. I'm not sure I have anything better than app. In either case app or system/os is I think clearer than registered.

For `Protocol` the thing we're launching is a URI not a protocol so maybe we should say 'Uri' instead?

* LaunchingAppUri
* LaunchingSystemUri
?

Reply:
LaunchingSystemUri makes the most sense to me, I'm going to go ahead and change the name to that if there's no objections to that.

Reply:
Actually it looks like there's public documentation that refers to this as ExternalProtocol, so naming it LaunchingExternalProtocol could potentiall work: https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-policies#externalprotocoldialogshowalwaysopencheckbox.  This ExternalProtocolDialog is only displayed for "protocols" that are registered with the OS.